lib/line-protocol.js

```diff
@@ -121,45 +93,41 @@ internals.values = {
                 {},
                 internals.formatError(event.data),
                 {
-                    id     : internals.String(event.id),
-                    method : internals.String(event.method && event.method.toUpperCase()),
-                    path   : internals.String(event.path),
-                    tags   : internals.String(event.tags)
+                    id     : Formatters.String(event.id),
+                    method : Formatters.String(event.method && event.method.toUpperCase()),
+                    path   : Formatters.String(event.path),
+                    tags   : Formatters.String(event.tags)
                 }
             );
         }
 
         return {
-            data   : internals.String(event.data),
-            id     : internals.String(event.id),
-            method : internals.String(event.method && event.method.toUpperCase()),
-            path   : internals.String(event.path),
-            tags   : internals.String(event.tags)
+            data   : Formatters.String(event.data),
+            id     : Formatters.String(event.id),
+            method : Formatters.String(event.method && event.method.toUpperCase()),
+            path   : Formatters.String(event.path),
+            tags   : Formatters.String(event.tags)
         };
     },
     response: (event) => {
 
         return {
-            httpVersion   : internals.String(event.httpVersion),
-            id            : internals.String(event.id),
-            instance      : internals.String(event.instance),
-            labels        : internals.String(event.labels),
-            method        : internals.String(event.method && event.method.toUpperCase()),
-            path          : internals.String(event.path),
-            query         : Qs.stringify(internals.String(event.query)),
```

Comment:
@fhemberger Is this order of operations correct?  Format as string first, then pass to `Qs.stringify`?  When we unit tested we found that it didn't format correctly.  We reversed the order (see below) and it seemed to format nicely.